python/seldon_core/seldon_methods.py

```diff
@@ -387,35 +441,55 @@ def route(
             client_response = client_route(
                 user_model, features, datadef.names, meta=meta
             )
+            client_response, runtime_metrics, runtime_tags = extract_runtime_data(
```

Comment:
Route is expected to return int so this wouldn't apply

Reply:
I thought we'd like users to be able to still define runtime metrics / tags on route method?

Reply:
ah, right, we have
```
    def route(
        self, features: Union[np.ndarray, str, bytes], feature_names: Iterable[str]
    ) -> int:
        raise SeldonNotImplementedError("route is not implemented")
```

okay, fair enough, removing from route

Reply:
Removed runtime metrics / tags from route method